.github/workflows/ossar-scan.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+# This workflow executes the OSSAR action.
+
+name: ossar-scan
+
+on:
+  workflow_call:
+    inputs:
+      # The name of the build job to wait for.
+      build_job:
+        required: true
+        type: string
+      # The name of the build artifact to download.
+      build_artifact:
+        required: true
+        type: string
+
+permissions:
+  checks: read  # Required by fountainhead/action-wait-for-check to wait for another GitHub check to complete.
+  contents: read  # Required by actions/checkout to fetch code.
+  security-events: write # Required by codeql task
+
+jobs:
+  scan:
+    timeout-minutes: 30
+
+    strategy:
+      matrix:
+        # For now only run on release as debug builds of the MSVC are built without /Qspectre
+        configurations: [Release]
+    runs-on: windows-latest
```

Comment:
Should this be `windows-2019`? We are using `windows-2019` at other places.

Reply:
The job doesn't run on Windows-2019 due to .NET issue. Only works on Windows-Latest:
https://github.com/github/ossar-action

Reply:
Maybe we can add that as a comment in yml.